### PR TITLE
Fixed typos; Some additions

### PR DESCRIPTION
**Pull request by KartikSoneji**
- Fixed typos.
- Removed Unix commands.
- Added some good use cases.
- Added section on `Command Redirection` and `Command Chaining`.

### CODE REVIEW

File: cmd.md [hunk 1]
```diff
@@ -96,34 +103,59 @@ OR
...
-- To write a command's output to a file, use `command > file_name.ext`
+- The `find` command will look for a word/phrase in a file or folder.  
+  Eg: `find "Hello" hello.txt` will print all the lines with the word `Hello` in the file `hello.txt`.
+- By default, the text is case sensitive (`Hello` is different from `hello`). Use the `/i` flag to ignore case.  
+  Eg: `find /i "Hello" hello.txt`.
+- You can also tell it to search all the files in the current folder with `find "Hello" *`, or another folder with `find "Hello" text\*`.
+
+## Redirecting output
+
+### Redirect to a file
```
Comment (HarshKapadia2) [inline cmd.md:126]: Please add the heading `(Writing to an existing file)` to `Redirecting to a file`.

File: cmd.md [hunk 2]
```diff
@@ -142,11 +174,20 @@ OR
...
+
+## Chaining commands
 
-- `color bt` (b = background colour, t = text colour)
-- For colour legend, type `color /?`.
-- Type `color` to reset all colours to default.
+- You can chain multiple commands using `&`, `&&` and `||`.
+- `command_1 & command_2` will run `command_1` and then run `command_2`.
+- `command_1 && command_2` will run `command_1` and then run `command_2` only if `command_1 finished successfully.
+- `command_1 && command_2` will run `command_1` and then run `command_2` only if `command_1 **did not** finished successfully.
```
Comment (HarshKapadia2) [inline cmd.md:191]: Check point number 3 and 4 of `chaining commands`, there seems to be some grammatical error and formatting errors.

Comment (KartikSoneji) [reply cmd.md]: Done.

**Comment by HarshKapadia2** [comment]
Thank you SO much for your contribution, @KartikSoneji! :smile:
Your work is appreciated!
I hope you consider contributing to this repo in the future as well!

**Pull request merged**